src/utils/ArticleCard.jsx

```diff
@@ -0,0 +1,12 @@
+const ArticleCard = ({ article }) => {
+  return (
+    <section className="ArticleCard">
+      <h3>{article.title}</h3>
+      <p>
+        created by {article.author} at {Date(article.created_at)}
+      </p>
+    </section>
```

Comment:
might be nice to see a bit more info here! 

leave this as something to address once you've completed all your tickets but it would be nice to have the date in a more normal, readable format

---

Does this file belong in `/utils`?

Reply:
Moved ArticleCard.jsx to components
I will be displaying more info on the list later down the line.